src/views/Simulation/SimulationViewHook.js

```diff
@@ -40,6 +40,8 @@ export const useSimulationView = () => {
   const [centerToPosition, setCenterToPosition] = useState(() => {});
 
   const [viewMode, setViewMode] = useState(DEFAULT_SETTINGS.viewMode);
+  const [selectedMapFilters, setSelectedMapFilters] = useState([]);
+  const [onlyCriticalPoints, setOnlyCriticalPoints] = useState(false);
```

Comment:
Do you think we could store the `onlyCriticalPoints` value in the `selectedMapFilters`? (it is a filter of the map view, so I suppose it would make sense 😄)
We could then extract it from the array in the `MapEntitiesDropdown` component, to keep a separate logic for the checkboxes vs. the toggle component for this option